apps/wallet-dashboard/components/dialogs/staking/views/DetailsView.tsx

```diff
@@ -53,7 +53,7 @@ export function DetailsView({
 }: StakeDialogProps): JSX.Element {
     const totalStake = BigInt(stakedDetails?.principal || 0n);
     const validatorAddress = stakedDetails?.validatorAddress;
-    const { isValidatorExpectedToBeInTheCommittee } =
+    const { isValidatorExpectedToBeInTheCommittee, isValidatorExpectedToBeInTheCommitteeLoading } =
```

Comment:
you assign `isLoading` to `isValidatorExpectedToBeInTheCommitteeLoading` if you do what I suggested about the rename